package.json

```diff
@@ -24,47 +23,44 @@
     "@material-ui/icons": "4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.56",
     "axios": "0.19.0",
-    "json-server": "0.15.1",
-    "npm-run-all": "4.1.5",
     "prop-types": "^15.7.2",
     "react": "16.11.0",
     "react-dom": "16.11.0",
     "react-hook-form": "5.5.1",
-    "react-query": "^2.5.14",
+    "react-query": "^2.15.4",
     "react-router-dom": "5.1.2",
     "react-scripts": "^3.4.0",
-    "react-select": "3.0.8",
-    "unfurl.js": "5.0.1"
+    "react-select": "3.0.8"
   },
   "devDependencies": {
-    "@babel/core": "^7.8.4",
-    "@babel/plugin-transform-runtime": "^7.9.0",
-    "@babel/preset-env": "^7.8.4",
+    "@babel/core": "^7.11.6",
+    "@babel/plugin-transform-runtime": "^7.11.5",
+    "@babel/preset-env": "^7.11.5",
     "@babel/preset-react": "^7.8.3",
     "@sheerun/mutationobserver-shim": "^0.3.3",
     "@storybook/addon-actions": "^5.3.9",
     "@storybook/addon-links": "^5.3.9",
     "@storybook/addons": "^5.3.9",
     "@storybook/preset-create-react-app": "^1.5.2",
     "@storybook/react": "^5.3.19",
```

Comment:
Storybook requires packages, which throw the following errors:

```
tsutils@3.17.1 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself.
react-docgen-typescript-loader@3.7.2 requires a peer of typescript@* but none is installed. You must install peer dependencies yourself.
react-docgen-typescript@1.20.4 requires a peer of typescript@>= 3.x but none is installed. You must install peer dependencies yourself.
```

We can install typescript@>= 3.x

Reply:
Same comment from above; Storybook isn't currently on my priority list as we're not using it in a very effective way, yet.

Reply:
Typescript is also a requirement of material-ui, which this repo heavily depends upon. This is the one library that will provide the most stability if it's added to this PR @angelocordon

Reply:
I'm not seeing anywhere in the Material UI docs that say we need Typescript in order to use Material UI. We've been able to use it this whole time without TS. I think perhaps you mean this [page](https://material-ui.com/guides/typescript/#typescript)? I believe this only means that if we do have static typing in our codebase, we'd need at least TS 3.2 for it to work with Material. 

Considering that this particular thread is about Storybook and we're now talking about Material, this is one of those opportunities where we can be much more practical and avoid some dependency pain points.